src/anjaliSDec19/Assignment2/MaxValueAtDiffIndex.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Find max value at differed index
+ */
+package anjaliSDec19.Assignment2;
+
+public class MaxValueAtDiffIndex {
+
+	void findMaxAtDifferedIndex(int[] arr1, int[] arr2)
+	{
+		if(arr1.length== arr2.length)
+		{
+			int cnt=0;
+			for(int i =0; i<arr1.length;i++)
+			{
+				if(arr1[i]!=arr2[i] && cnt< 1)
+				{
+					cnt++;
+					System.out.println("Values are not matching at index : " + i);
+					if(arr1[i]>arr2[i])
```

Comment:
here you need to find only index not to check greater and smaller value. but it fine it good, you able to think out of box.